pool/app/matcher/matcher.ml

```diff
@@ -0,0 +1,68 @@
+let get_or_failwith = Pool_common.Utils.get_or_failwith
+
+let count_of_rate interval rate =
+  (* calculated number of invitations from the rate per hour -> runns every
+     <interval> minutes *)
+  if interval > 60 || interval < 1
+  then Result.error Pool_common.Message.(Invalid Field.Interval)
+  else
+    CCFloat.(of_int (Mailing.Rate.value rate) / 60.0 * of_int interval |> round)
+    |> CCInt.of_float
+    |> Result.ok
+;;
+
+let i18n_templates pool { Experiment.invitation_template; _ } languages =
+  let open Utils.Lwt_result.Infix in
+  let open Experiment.InvitationTemplate in
+  let open I18n in
+  Lwt_list.map_s
+    (fun language ->
+      let%lwt subject =
+        find_by_key pool Key.InvitationSubject language ||> get_or_failwith
+      in
+      let%lwt text =
+        find_by_key pool Key.InvitationText language ||> get_or_failwith
+      in
+      let subject =
+        invitation_template
+        |> CCOption.map_or ~default:subject (fun { subject; _ } ->
+             subject
+             |> Subject.value
+             |> Content.of_string
+             |> create Key.InvitationSubject language)
+      in
+      let text =
+        invitation_template
+        |> CCOption.map_or ~default:text (fun { text; _ } ->
+             text
+             |> Text.value
+             |> Content.of_string
+             |> create Key.ImportInvitationText language)
+      in
+      (language, (subject, text)) |> Lwt.return)
+    languages
+;;
+
+let find_contacts_by_mailing
+  pool
+  ?(interval = 5)
```

Comment:
Isn't the interval defined by how often the command runs? So there will be no way to change that?